src/pages/getting-started/hello-world.mdx

```diff
@@ -145,9 +145,19 @@ await Deno.writeTextFile("key.addr", address);
   and type/copy each line.
 </Callout>
 
-Now, we can head to [the Cardano faucet](https://docs.cardano.org/cardano-testnet/tools/faucet) to get some funds on the preview network to our newly created address (inside `key.addr`). Make sure to select _"Preview Testnet"_ as network.
+<Callout type="info" emoji="ℹ️">
+    If you have an existing (Private) Signing Key that you generated with `cardano-cli` or `cardanocli-js` and you prefer to use that instead, then make sure to first convert the CborHex to a Bech32 encoded key hash. You can do this in the terminal by running:
 
-Using [CardanoScan](https://preview.cardanoscan.io/) we can watch for the faucet sending some ADA our way. This should be pretty fast.
+  ```
+  echo <REPLACE_WITH_PRIVATE_KEY_CBORHEX> | bech32 ed25519_sk > skey_bech32
+  ```
+
+  Make sure to remove the first two bytes(a "5820" prefix in most cases) in the CborHex before proceeding with this. 
+</Callout>
+
+Now, we can head to [the Cardano faucet](https://docs.cardano.org/cardano-testnet/tools/faucet) to get some testnet funds on the Preview network to our newly created address (inside `key.addr`). Make sure to select _"Preview Testnet"_ as network.
+
+Using [CardanoScan](https://preview.cardanoscan.io/) we can watch for the faucet sending some tADA our way. This should be pretty fast.
```

Comment:
I'd rather avoid the multiplication of tools and and oddities in this tutorial. We purposely avoid any command-line tool here to make the setup easy. There are plenty of other resources about how to use those tools, so it isn't the scope of this tutorial. I am afraid it will only adds confusion.

Please revert all the changes between L147 and L160.